.vscode/launch.json

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
We can git ignore this file, it seems pretty generic and IDE specific. If everyone working on a project added their IdE configuration files, we'd quickly have a large number of IDE configuration files.

Reply:
The important part here is `"outputCapture": "std"` as this needs to be manually added, the rest is auto generated. Could be on a wiki page.